package.json

```diff
@@ -5,15 +5,19 @@
   "main": "dist/server.bundle.js",
   "scripts": {
     "bootstrap": "shx rm -rf dist && shx mkdir dist",
-    "start": "npm run build && run-p server watch:**",
+    "start": "npm run build:dev && run-p server watch:**",
     "build": "run-s bootstrap build:**",
-    "heroku-postbuild": "echo Skip builds on Heroku",
+    "heroku-postbuild": "npm run build:prod",
```

Comment:
Is this a safe change? The idea of heroku-postbuild is that it triggers _after_ a build, not for it to trigger a new build =x

Reply:
ah I followed what we have for foundation site but will revert this and file changes as a followup instead